theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1437,7 +1454,7 @@ def test_basic(self):
         # test that function contains softmax and no div.
         f = theano.function([c], p_y, mode=self.mode)
 
-        assert hasattr(f.maker.fgraph.outputs[0].tag, 'trace')
+        assert check_stack_trace(f, ops_to_check=softmax_op)
```

Comment:
Indeed, `copy_stack_trace` is not called from [`softmax_simplifier`](https://github.com/lamblin/Theano/blob/fix_stack_check_nnet/theano/tensor/nnet/nnet.py#L878-L913).